scripts/maven-release-publisher.yml

```diff
@@ -0,0 +1,38 @@
+version: 0.2
+env:
+  shell: /bin/sh
+  secrets-manager:
+    ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: amplify/android/sonatype:username
+    ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: amplify/android/sonatype:password
+    ORG_GRADLE_PROJECT_signingPassword:  amplify/android/signing:password
+    ORG_GRADLE_PROJECT_signingKeyId:  amplify/android/signing:keyId
+    ORG_GRADLE_PROJECT_signingInMemoryKey: amplify/android/signing:inMemoryKey
+    ORG_GRADLE_PROJECT_signingSecretKeyRingFile:  amplify/android/signing:secretKeyRingFile
```

Comment:
If the `inMemoryKey` is working, can we get rid of the `secretKeyRingFile`?  Isn't it intended to be used _in place of_ `secretKeyRingFile`?

Reply:
Yes...I left it in there for now, but we can get rid of it once we get it working with the inMemoryKey